src/components/input/math-wrapper.js

```diff
@@ -7,12 +7,12 @@
 const $ = require('jquery');
 // Keeping `window` in place for test suite and GitHub Pages.
 // If it does not exist, fall back to CommonJS require. - jsatk
-const MathQuill = window.MathQuill || require('mathquill');
+const MathQuill = require('mathquill');
 
 const Keys = require('../../data/keys');
 const CursorContexts = require('./cursor-contexts');
-const {DecimalSeparators} = require('../../consts');
-const {decimalSeparator} = require('../../utils');
+const { DecimalSeparators } = require('../../consts');
```

Comment:
Any particular reason for the spaces here? We don't use this pattern in webapp.

Reply:
We need to get this project under prettier ASAP. I'll add a ticket for that too. We're just fighting the VSCode formatter here- it totally wasn't intentional

Reply:
Okay cool, thanks for the explanation!